.tekton/scanner-component-pipeline.yaml

```diff
@@ -298,14 +275,16 @@ spec:
       value: $(tasks.build-container.results.IMAGE_URL)
     - name: BASE_IMAGES
       value: $(tasks.build-container.results.BASE_IMAGES_DIGESTS)
-    runAfter:
-    - build-container
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
```

Comment:
Could/should this be `tasks.fetch-scanner-data.results.SOURCE_ARTIFACT` as the last task that edited the source?
(it won't matter for what this task attempts to achieve, nor for the order of tasks, just for the next person to copy&paste the params from this task to a latter task)

Reply:
I've also switched `sast-snyk-check` to the `fetch-scanner-data` source artifact for consistency